.github/workflows/superset-python-integrationtest.yml

```diff
@@ -80,7 +80,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [3.8, 3.9]
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
Without quotes, YAML interprets `3.10` as a number i.e., `3.1` and thus I felt it was prudent to explicitly define all versions as strings and not numbers.